src/core/main.c

```diff
@@ -72,6 +72,9 @@
 #include "process-util.h"
 #include "raw-clone.h"
 #include "rlimit-util.h"
+#ifdef HAVE_SECCOMP
```

Comment:
Because seccom-util.h is not part of the dist if HAVE_SECCOMP is not defined (https://github.com/systemd/systemd/blob/v231/Makefile.am#L1065). I think this would break distcheck?